deploy/template.yaml

```diff
@@ -1065,6 +1065,50 @@ Resources:
         - AddProvisionedConcurrency
         - ProvisionedConcurrentExecutions: !FindInMap [ EnvironmentConfiguration, !Ref AWS::AccountId, provisionedConcurrency ]
         - !Ref AWS::NoValue
+        -
+  SelectCriFunction:
```

Comment:
Could we move this Function down below the log groups of the Gpg45Evaluator ones.